examples/common_vmseries/variables.tf

```diff
@@ -900,6 +903,22 @@ variable "vmseries" {
     The `private_snet_key` and `public_snet_key` are required when `bootstrap_xml_template` is set.
     EOF
   }
+  validation {
+    # Ensure only one of `application_gateway_key` or `appgw_backend_pool_id` is set under an interface.
```

Comment:
This comment is not necessary, you already state it in the error message. The comment pattern we follow for validation blocks is like this (names of the properties you're validating):
```hcl
validation { # bootstrap_options & bootstrap_package
(...)
}
```

Reply:
Good point - removed!